riscv_isac/main.py

```diff
@@ -105,9 +105,16 @@ def cli(verbose):
         help = "Coverage labels to consider for this run."
 )
 @click.option('--xlen','-x',type=click.Choice(['32','64']),default='32',help="XLEN value for the ISA.")
+@click.option(
+        '--list-duplicate',
+        '-z',
+        is_flag='True',
+        default=False,
+        help='Enable to print out the duplicate coverpoints generated'
+    )
 def coverage(elf,trace_file, window_size, cgf_file, detailed,parser_name, decoder_name, parser_path, decoder_path,output_file, test_label,
-        sig_label, dump,cov_label, xlen):
-    isac(output_file,elf,trace_file, window_size, expand_cgf(cgf_file,int(xlen)), parser_name, decoder_name, parser_path, decoder_path, detailed, test_label,
+        sig_label, dump,cov_label, xlen, list_duplicate):
+    isac(output_file,elf,trace_file, window_size, expand_cgf(cgf_file,int(xlen),list_duplicate,cov_label), parser_name, decoder_name, parser_path, decoder_path, detailed, test_label,

```

Comment:
The `isac` function has not been updated to support this functionality.

Reply:
`isac` function uses the expanded cgf only, so there is no need for changes in that function right? The change is made only in call to expand_cgf, which has been modified.